### ISSUE TITLE
info print out to screen

### ISSUE THREAD

**Issue by wenbinyu-PU** [wenbinyugroup/ivabs#50]
Su, I think we provided too much info to the screen which slows down the running of the code. Since most of the info can basically read, I will suggest to shorten the info or write such info to a file. Only keep the key info.

**Comment by unitasium** [comment]
Indeed. I will later better organize information into different levels like 'critical, warning, info, debug'.
